playbooks/deploy_edpm_os_install.yml

```diff
@@ -56,9 +56,21 @@
         tasks_from: install.yml
       tags:
         - edpm_chrony
+    - name: install frr
+      ansible.builtin.import_role:
+        name: osp.edpm.edpm_frr
+        tasks_from: install.yaml
+      tags:
+        - edpm_frr
     - name: Install edpm_ovn
       ansible.builtin.import_role:
         name: osp.edpm.edpm_ovn
         tasks_from: install.yml
       tags:
         - edpm_ovn
+    - name: install ovn_bgp_agent
```

Comment:
Task name should be start with an uppercase letter (see https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/rules/name.py#L127)